src/checks/requiredTypeScript.ts

```diff
@@ -1,14 +1,15 @@
 import * as core from '@actions/core'
 import * as CommentedJSON from 'comment-json'
-
 import { WebhookEventMap } from '@octokit/webhooks-types'
+import ignore, { Ignore as IgnoredFileFilter } from 'ignore'
 
 import { github } from '../infrastructure/github'
 import * as fs from '../infrastructure/fs'
 import { isBot } from '../conditions/triggeredByBot'
 
 import Check from './check'
-import ignore, { Ignore as IgnoredFileFilter } from 'ignore'
+
+const JS_TS_CHECK_COMMENT = '// @ts-check'
```

Comment:
Asking out of ignorance: is the whitespace optional here? e.g. would `//@ts-check` be accepted by `tsc`?

Reply:
Good catch. I'll change it to a regex that accepts any amount of whitespace 👍

Reply:
Great!